### ISSUE TITLE
Disabled Selectmenu receives focus on tab

### ISSUE THREAD

**Issue by gcko** [fnagel/jquery-ui#136]
You can tab to a disabled selectmenu and still change its values. I can submit a pull request to fix this, not much to it though -- just need to check to see if the selectmenu is in a disabled state and return out of any events

**Comment by fnagel** [comment]
I simply added a if statement to the moveSelection method, see latest commit.

Please test and give feedback.

**Comment by fnagel** [comment]
Please give feedback!

**Comment by gcko** [comment]
The selection values are not modifiable now, however you can still tab to a disabled selectmenu. A disabled widget should be outside of the tab order.

**Comment by fnagel** [comment]
You're right. Thanks for reporting. Fixed in latest commit!

**Comment by gcko** [comment]
Cool, I'll test as well :)
